examples/timestamping/backend/src/lib.rs

```diff
@@ -42,17 +49,12 @@ use schema::Schema;
 use transactions::TimeTransactions;
 
 const TIMESTAMPING_SERVICE: u16 = 130;
-pub const SERVICE_NAME: &str = "timestamping";
+const SERVICE_NAME: &str = "timestamping";
 
+/// Demo timestamping service.
```

Comment:
Not sure whether I understand the meaning here. Does this code launch the service? Perhaps it would be better to rephrase the sentence, to make the meaning clearer.